Header header
string[] sensor_model
float32[] reading
float32 temperature_c		# temperature in celsius
float32 temperature_f		# temperature in fahrenheit
float32 relative_humidity	# humidity in %

